config/dev.js

```diff
@@ -21,22 +23,23 @@ module.exports = {
         apiDevHost,
       ],
       fontSrc: [
-        `${addonsFrontendCDN}/static/`,
+        `${baseUrlDev}${staticPath}`,
       ],
       imgSrc: [
         "'self'",
         'data:',
-        addonsServerDevCDN,
-        `${addonsFrontendCDN}/static/`,
+        `${addonsServerDevCDN}/user-media/`,
+        `${addonsServerDevCDN}/static/`,
+        `${baseUrlDev}${staticPath}`,
         // This file isn't bundled with addons-frontend.
-        `${addonsFrontendCDN}/favicon.ico`,
+        `${addonsServerDevCDN}/favicon.ico`,
```

Comment:
This was wrong. We currently load the favicon from addons-server CDN, it 404s on addons-frontend CDN.